### DISCUSSION TITLE
How can I focus the cursor on the text input field added on row N - cell 1, when a new row is added? (Text input control is added using Cell render function.)

### DISCUSSION THREAD

**Discussion by danvln** [TanStack/table#2800] [Q&A]
Hey folks, 

I cannot find the references exposed. Probably, I am missing something, since I am new to this lib. 

What I try to do is to focus the cursor on the input type that I have added on the first cell of a row, let's say row N = 10. (How can I focus the cursor on the text input field added on row N - cell 1, when a new row is added? (Text input control is added using Cell render function.))

Thanks in advance for any help!

**Comment by danvln** [comment]
Anyone?

**Reply by gargroh** [reply]
Code sandbox url of the same please
